shard_row_id_bits.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Please:
1. rename this file to shard-row-id-bits.md.
2. Add this file to TOC.md. 

@kissmydb